_spinnaker_install_admin_guides/install-on-aws-halyard-pod.md

```diff
@@ -0,0 +1,668 @@
+---
+layout: post
+title: Installing Spinnaker in AWS (with Halyard in-cluster)
+order: 22
+published: true
+redirect_from:
+  - /spinnaker_install_admin_guides/install_on_eks-halyard-pod/
+  - /spinnaker_install_admin_guides/install-on-eks-halyard-pod/
+  - /spinnaker-install-admin-guides/install_on_eks_halyard_pod/
+  - /spinnaker-install-admin-guides/install-on-eks_halyard_pod/
+  - /spinnaker-install-admin-guides/install-on-aws-halyard-pod/
+  - /spinnaker_install_admin_guides/install_on_aws_halyard_pod/
+  - /spinnaker_install_admin_guides/install-on-aws-halyard-pod/
+---
+
+This guide describes how to install Spinnaker in AWS or in an on-prem Kubernetes cluster with access to S3. It will create and use the following Amazon Web Services resources:
+
+- A Kubernetes cluster running on Amazon Web Services (AWS). EKS is a good way to get a Kubernetes cluster up on AWS - see the AWS documentation for this.
+- An Amazon S3 (Simple Storage Service) bucket. You can use an existing one or create a new one.
+- An NGINX Ingress controller in your AWS cluster.
```

Comment:
AWS *EKS* cluster?  (I think to help disambiguate for newbs. )

Reply:
done